cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go

```diff
@@ -92,25 +86,20 @@ func Run(ctx context.Context, options *resolveroptions.Options) error {
 		logger.Error(err, "failed to create config from flags")
 		return err
 	}
-	edgeViewConfig, err := configForViewOfExport(ctx, espwRestConfig, "edge.kubestellar.io")
+	espwClientset, err := edgeclientset.NewForConfig(espwRestConfig)
 	if err != nil {
-		logger.Error(err, "failed to create config for view of edge exports")
+		logger.Error(err, "failed to create config for edge exports")
```

Comment:
"edge exports" seems not quite on-point for kube-bind. This is for access to the consumer API object copies in the service provider's space, right?

Reply:
Changed